### PR TITLE
Inject seo resolver

### PR DESCRIPTION
**Pull request by hlohaus**
### 1. Why is this change necessary?

Make it easer to extend it.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Storefront/Framework/Routing/RequestTransformer.php [hunk 1]
```diff
@@ -47,10 +47,16 @@ class RequestTransformer implements RequestTransformerInterface
...
 
-    public function __construct(RequestTransformerInterface $decorated, Connection $connection)
+    /**
+     * @var SeoResolverInterface
+     */
+    private $resolver;
+
+    public function __construct(RequestTransformerInterface $decorated, Connection $connection, SeoResolverInterface $resolver)
     {
         $this->connection = $connection;
```
Comment (mitelg) [inline src/Storefront/Framework/Routing/RequestTransformer.php:57]: seems obsolete now

**Comment by mitelg** [comment]
hey @hlohaus 
thanks for your contribution, but please fill out the PR template correctly :wink:

File: src/Storefront/Framework/Routing/RequestTransformer.php (hunk 1 above)
Comment (hlohaus) [reply src/Storefront/Framework/Routing/RequestTransformer.php:57]: findSalesChannel use it too

**Comment by mitelg** [comment]
thanks again @hlohaus :+1: :tada: :blue_heart:

**Pull request closed without merging**